src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -1214,74 +1215,77 @@ private void updateServerStatusIconAndText(RemoteOperationResult result) {
         mServerStatusIcon = R.drawable.common_error;    // the most common case in the switch below
 
         switch (result.getCode()) {
-        case OK_SSL:
-            mServerStatusIcon = R.drawable.ic_lock;
-            mServerStatusText = R.string.auth_secure_connection;
-            break;
-
-        case OK_NO_SSL:
-        case OK:
-            if (mHostUrlInput.getText().toString().trim().toLowerCase().startsWith("http://") ) {
-                mServerStatusText = R.string.auth_connection_established;
-                mServerStatusIcon = R.drawable.ic_ok;
-            } else {
-                mServerStatusText = R.string.auth_nossl_plain_ok_title;
+            case OK_SSL:

```

Comment:
This update of indentation, was on purpose or triggered by automatic formatting in your AS? Please, **disable** it if that's the case.